doc/man3/BN_generate_prime.pod

```diff
@@ -51,7 +51,9 @@ L<openssl_user_macros(7)>:
 =head1 DESCRIPTION
 
 BN_generate_prime_ex() generates a pseudo-random prime number of
-at least bit length B<bits>.
+at least bit length B<bits>. The returned number is probably prime
```

Comment:
I'd either omit _probably_ or omit _with a negligible error probability_.  There is no necessity for both.

Reply:
The [original formulation](https://github.com/openssl/openssl/pull/9257/commits/5cec22604b830f80f05148d61c8964610ccb7750#diff-c4d2f8e93c2462d9cedf7f21b30b9158L93) (which I moved upwards, because it did not fit there) was a bit unspecific. That's why I added _probably prime_ as a technical term which everybody understands.

So I'd rather keep _probably prime_.  Instead of removing _with a negligible error probability_, how about just removing the single word _probability_?

```
The returned number is probably prime with a negligible error.
```

Reply:
See e6facdfeb5fd905e78c260edb6402c88f28302f2.